lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -93,7 +93,7 @@ class Server {
     const filePath = requestUrl.pathname;
     const queryString = requestUrl.search && parseQueryString(requestUrl.search.slice(1));
     let absoluteFilePath = path.join(this.baseDir, filePath);
-    const sendResponse = (statusCode, data) => {
+    const sendResponse = (statusCode, data, extraHeaders) => {
```

Comment:
nit: could you call this something like `baseHeaders` so it isn't confused with the URL supplied `extraHeaders`